plugins/modules/inventory_intent.py

```diff
@@ -828,36 +840,68 @@ def export_device_details(self):
 
                     return self
 
-            # With this File ID call the Download File by FileID API
+            # With this File ID call the Download File by FileID API and process the response
             response = self.dnac._exec(
                 family="file",
                 function='download_a_file_by_fileid',
                 op_modifies=True,
                 params={"file_id": file_id},
             )
 
-            device_data = []
-            encoded_resp = response.data.decode(encoding='utf-8')
-            self.log(str(encoded_resp))
+            if payload_params["operationEnum"] == "0":
+                zip_data = BytesIO(response.data)
+
```

Comment:
can we write, 
if not HAS_PYZIPPER:
and return from here itself?

Reply:
added.